Client/Frontend/Browser/TopTabCell.swift

```diff
@@ -0,0 +1,343 @@
+//
+//  TopTabCell.swift
+//  Client
+//
+//  Created by Tyler Lacroix on 6/27/16.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
+
```

Comment:
The file name doesn't really match whats going on in here. Might make more sense to name this 'TopTabViews' and extract the collection view delegate/datasource into their own file or files.